src/test/resources/features/ifs/DebtCalculationValidation.feature

```diff
@@ -38,7 +38,7 @@ Feature: Debt Calculation Validation
 #      | originalAmount | dateCreated | interestStartDate | interestRequestedTo | mainTrans | subTrans | interestBearing |
 #      | 1.2            | 2021-03-01  | 2021-03-01        | 2021-03-08          | 1525      | 1000     | true            |
 #    And the debt item has no payment history
-#    And no breathing spaces have been applied to the customer
+#    And no breathing spaces have been applied to the debt item
```

Comment:
This is all commented out

Reply:
commented out steps not part of my changes - one step defination changed